Added lroc calc_pushbroom_keywords app

cordellmichaud opened the pull request:
Added lroc calc_pushbroom_keywords app

## Description
I added the calc_pushbroom_keywords app to the lro/apps directory.

## Related Issue
N/A

## Motivation and Context
The calc_pushbroom_keywords app is needed to run on a NAC cube as part of image preprocessing.

## How Has This Been Tested?
In the ISIS 3.10.2_lroc conda evironment, calc_pushbroom_keywords was run on a NAC cube after lronac2isis and spiceinit, resulting in a .lis file. This file was compared to an expected .lis keywords file to ensure the values of keywords did not differ significantly.

## Screenshots (if appropriate): N/A

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My code follows the code style of this project.
- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

cordellmichaud committed:
Added calc_pushbroom_keywords app

victoronline commented on isis/src/lro/apps/calc_pushbroom_keywords/get_mounting_angles.cpp:
```diff
@@ -0,0 +1,156 @@
+*       Jan 28 2010 EHK - In order to determine LRO backwards-aquired images  *
+*                         removed 180 degree adjustment to kappa from this    *
+*                         routine and placed it in calc_pushbroom_keywords.   *
+*                                                                             *
```
Cordell, add comments in the history section denoting the change.

cordellmichaud replied:
For which change am I commenting history? That it was re-added to the lro apps and calc_pushbroom_keywords.cpp was renamed? I didn't end up needing to rewrite anything since it built properly without change. 

Though I did find a change in Jan 2020 to calc_pushbroom_keywords.cpp/main.cpp in svn log about changing ephemera to .25 that isn't accounted for in the history section.

victoronline replied:
Comment on re-adding it to run in the 3.10.2_lroc version. You can also mention the ephemera change that was made. Make sure you mention who it was made by. "Change to ... made by @someone on some date. Since its "new" to this version, it would be good to make sure all is well documented.

cordellmichaud replied:
Understood. I'll add that now.

cordellmichaud replied:
I have updated the history sections with the changes, names and dates.

cordellmichaud committed:
Update history section with changes

- Added history note about change to ephemera value
- Added history entry about re-adding to run in the 3.10.2_lroc version of ISIS

cordellmichaud committed:
Update history section of get_naif_matrices.cpp

- Added history entry about re-adding to run in the 3.10.2_lroc version of ISIS

cordellmichaud committed:
Update history section of get_mounting_angles.cpp

- Added history entry about re-adding to run in the 3.10.2_lroc version of ISIS

cordellmichaud committed:
Update history section of calc_pushbroom_keywords.xml

- Added history entry about re-adding to run in the 3.10.2_lroc version of ISIS
